modules/products/bamboo/helm.tf

```diff
@@ -7,7 +7,7 @@ resource "helm_release" "bamboo" {
   repository = local.helm_chart_repository
   chart      = local.bamboo_helm_chart_name
   version    = local.bamboo_helm_chart_version
-  timeout    = 40 * 60 # dataset import can take a long time
+  timeout    = 10 * 60
```

Comment:
Will 10mins be enough to import dataset?

Reply:
is more than enough... for Bamboo it takes 3 minutes max